ClinGen/clincoded
# Connects to #1103 update modes of inheritance for GCI/VCI

Updates modes of inheritance in gdm and interpretation objects: schemas updates, upgrade scripts, and UI changes.

**To test:**
1. Check dropdowns in UI:
 - GCI-> +new GDM [check mode of inheritance dropdown]
 - VCI -> +variant -> +Interpretation -> +Mode of Inheritance [check dropdown]
2. check all gdms page (http://curation.clinicalgenome.org/gdm -> 
`X-linked recessive inheritance (HP:0001419)` is now `X-linked inheritance (HP:0001417)`)

**Note:** Updates one identified mode in 1 gdm in production and several others in test data

**Note:** **New GCI/VCI allowed modes of inheritance:**
   "Autosomal dominant inheritance (HP:0000006)",
    "Autosomal recessive inheritance (HP:0000007)",
    "Unknown",
    "Mitochondrial inheritance (HP:0001427)",
    "Multifactorial inheritance (HP:0001426)",
    "Somatic mutation (HP:0001428)",
    "X-linked inheritance (HP:0001417)",
    "Y-linked inheritance (HP:0001450)",
    "Other"
